symplyphysics/laws/kinematic/vector/acceleration_relative_to_reference_frame.py

```diff
@@ -0,0 +1,87 @@
+from sympy import Expr, symbols
+from symplyphysics import (
+    units,
+    validate_input,
+    validate_output,
+    Vector,
+    Quantity,
+    QuantityVector,
+    add_cartesian_vectors,
+    subtract_cartesian_vectors,
+    scale_vector,
+)
+from symplyphysics.core.vectors.arithmetics import (
+    diff_cartesian_vector,
+    integrate_cartesian_vector,
+)
+
+# Description
+## For any reference frame, whether it is inertial or not, the motion relative to it can be described using
+## the position vector relative to that frame's origin.
+
+# Law: a_rel = (d**2(x)/dt**2, d**2(y)/dt**2, d**2(z)/dt**2)
```

Comment:
Why not `dv / dt`? I think it's shorter and more consistent with scalar version of this law.

Reply:
Then it should have the limit `dt -> 0`.

Reply:
Isn't it a definition of `dt`?